charts/enterprise-kyverno-operator/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: nirmata-kyverno-operator
 description: Helm Chart for Enterprise Kyverno Operator
 type: application
 version: v0.3.5
-appVersion: v0.2.0
+appVersion: v0.1.19
```

Comment:
Should be 0.2.1

Reply:
@pns-nirmata keeping this unchanged as discussed over call, unless you recommend otherwise.

Reply:
Correct. Should be unchanged.